setup.py

```diff
@@ -68,9 +79,7 @@ def run(self):
     author='HeikoHeiko',
     author_email='heiko@brainbot.com',
     url='https://github.com/raiden-network/raiden',
-    packages=find_packages(
-        exclude=["raiden.tests", "raiden.tests.*"]
-    ),
+    packages=find_packages(),
```

Comment:
The only thing I don't feel too comfortable with is including also test packages in the release. I understand the reasoning behind it, since we want it to use in the smoke tests but still I feel a bit uncomfortable with it.